Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -436,6 +436,56 @@ u32 GetXFBAddressBottom()
 		return m_XFBInfoBottom.FBB;
 }
 
+float GetAspectRatio(bool wide)
+{
+	int height = (2 * m_VerticalTimingRegister.ACV);
```

Comment:
Got it.  It's weird, because I tested gamecube games in progressive mode and it worked fine.  Guess the wii does things a little differently